proposed/cache.md

```diff
@@ -315,6 +303,20 @@ interface CacheItemPoolInterface
     public function getItems(array $keys = array());
 
     /**
+     * Confirms if the cache item exists in the cache.
+     *
+     * Note: This method MAY avoid retrieving the cached value for performance
+     * reasons, which could result in a race condition between exists() and get().
+     * To avoid that potential race condition use isHit() instead.
```

Comment:
The docblock also needs to be updated to clarify that we're talking about get() on Item, not on Pool.